### PR TITLE
Bump `taskkill` and use `execa` instead of `pify`

### PR DESCRIPTION
**Pull request by kevva**
Merge when https://github.com/sindresorhus/taskkill/pull/3 is merged and a new version is released.

### CODE REVIEW

**Comment by sindresorhus** [comment]
`taskkill` published.

**Comment by sindresorhus** [comment]
Can you ES2015ify the code too? I'm gonna require Node.js >=4. Also don't need the promise ponyfill then.

**Comment by kevva** [comment]
Yup, done.

**Comment by sindresorhus** [comment]
You have to remove the old Node.js versions in .travis.yml too.

File: index.js [hunk 1]
```diff
@@ -1,46 +1,41 @@
...
-	var fn = process.platform === 'win32' ? win : def;
-	var errors = [];
+	const fn = process.platform === 'win32' ? win : def;
+	const errors = [];
 
 	// don't kill ourselves
-	input = arrify(input).filter(function (el) {
-		return el !== process.pid;
-	});
+	input = arrify(input).filter(el => el !== process.pid);
```
Comment (sindresorhus) [inline index.js]: I usually use `x` instead of `el` for inline arrows fns like this.

**Comment by sindresorhus** [comment]
And add `"xo": {"esnext": true}` to package.json.

**Comment by sindresorhus** [comment]
Woot! :)

**Pull request merged**